### PR TITLE
BGDIINF_SB-3194: Added import maps results

### PR DESCRIPTION
**Pull request by ltshb**
[Test link](https://sys-map.dev.bgdi.ch/preview/feat-bgdiinf_sb-3194-import-tool-ui-p5/index.html)

### CODE REVIEW

**Review by pakb — APPROVED** [review]

File: src/api/layers/WMSCapabilitiesParser.class.js [hunk 1]
```diff
@@ -103,7 +103,16 @@ export default class WMSCapabilitiesParser {
...
+        )
+            .filter((layer) => !!layer)
+            .sort((layerA, layerB) => {
+                if (layerA.name.toLowerCase() > layerB.name.toLowerCase()) {
+                    return 1
+                } else if (layerA.name.toLowerCase() < layerB.name.toLowerCase()) {
+                    return -1
+                }
+                return 0
+            })
```
[the remark this answers is not in the document]
Comment (ltshb) [reply src/api/layers/WMSCapabilitiesParser.class.js:115]: 👍🏼  thanks I changed it and also moved into the ImportCatalogue component which is the display component. This solve also the sorting for the WMTS which was not done

File: src/modules/menu/components/LayerCatalogueItem.vue [hunk 2]
```diff
@@ -73,56 +76,90 @@ function startLayerPreview() {
...
     }
 }
+
+function onCollapseClick() {
+    showChildren.value = !showChildren.value
+}
+
+function onCheckboxClick() {
+    addLayer()
+}
```
[the remark this answers is not in the document]
Comment (ltshb) [reply src/modules/menu/components/LayerCatalogueItem.vue]: done

File: src/api/layers/WMSCapabilitiesParser.class.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ltshb) [reply src/api/layers/WMSCapabilitiesParser.class.js]: done

**Comment by ltshb** [comment]
@pakb  I implemented almost all what we discus on our call, see below

- [ ] remove connect
- [ ] open provider -> focus input
- [ ] change checkbox and plus 
  - [ ] here I've also changed the icon to be discussed tomorrow with the team, I've in comment 3 solutions for the icons.
- [ ] clear open providers and focus input 
- [ ] click outside margin
- [ ] attribution open localhost bug in legend 
- [ ] close import maps v-show to keep result
- [ ] reload capabilities on language change

For the last one I would like to discuss it again tomorrow
- [ ] pristine = true -> pristine = false on first change, error only if not pristine

**Comment by ltshb** [comment]
@pakb could you review my latest changes ?

File: tests/e2e-cypress/integration/importTool.cy.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ describe('The Import Maps Tool', () => {
         cy.goToMapView({}, true)
         cy.clickOnMenuButtonIfMobile()
     })
-    it('Import external wms layers', () => {
```
[the remark this answers is not in the document]
Comment (ltshb) [reply tests/e2e-cypress/integration/importTool.cy.js]: oops

**Pull request merged**